ta-sdk-sample/src/main/java/com/ibm/ta/sdk/sample/SamplePluginProvider.java

```diff
@@ -40,6 +40,11 @@
   private static final String FILE_ASSESS_CONFIG_FILE_XML= "/sampleData/sampleData.xml";
   private static final String FILE_ASSESS_CONFIG_FILE_XML2= "/sampleData/Plants.ear_server.xml";
 
+  @Override
+  public String getVersion() {
+    return "0.9.1";
```

Comment:
If this is the first time we are giving the sample a version can we set it to 0.5.1 or similar to give us breathing room before we go to 1.0.0 (aka first GA)